tests/ballerina-spec-conformance-tests/src/test/java/org/ballerinalang/test/TestRunnerUtils.java

```diff
@@ -196,6 +199,36 @@ private static String validateKindOfTest(String kind) {
         }
     }
 
+    public static void validateLabels(String labels, Set<String> selectedLabels) {
+        List<String> labelsList = new ArrayList<>();
+        StringJoiner duplicateLabels = new StringJoiner(", ");
+        StringJoiner unknownLabels = new StringJoiner(", ");
+
+        for (String label : labels.split(",")) {
+            String trimmedLabel = label.trim();
+            if (labelsList.contains(trimmedLabel)) {
+                duplicateLabels.add(trimmedLabel);
+                continue;
+            }
+            if (!selectedLabels.contains(trimmedLabel)) {
+                unknownLabels.add(trimmedLabel);
+            }
+            labelsList.add(trimmedLabel);
+        }
+
+        boolean hasUnknownLabels = unknownLabels.length() != 0;
+        boolean hasDuplicateLabels = duplicateLabels.length() != 0;
```

Comment:
Length cannot be negative. Lets use the exact logical expression.
```suggestion
        boolean hasUnknownLabels = unknownLabels.length() > 0;
        boolean hasDuplicateLabels = duplicateLabels.length() > 0;
```